### PR TITLE
suffix identifiers in backticks with colons consistently

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

File: format/index.md [hunk 1]
```diff
@@ -261,9 +261,9 @@ relationship isn't currently associated with any target resources.
 
 Resource linkage **MUST** be represented as one of the following:
 
-* `null` for empty to-one relationships.
-* a [resource identifier object] for non-empty to-one relationships.
-* an empty array (`[]`) for empty to-many relationships.
+* `null`: for empty to-one relationships.
```
Comment (dgeb) [inline format/index.md]: I don't think we want a `:` here, since we're not defining anything.

File: format/index.md [hunk 2]
```diff
@@ -261,9 +261,9 @@ relationship isn't currently associated with any target resources.
...
+* `[]`: for empty to-many relationships.
```
Comment (dgeb) [inline format/index.md]: Same here - probably ok to stick with the old wording about "an empty array (`[]`)"

**Comment by dgeb** [comment]
Looks good except for those two nits.

**Comment by tkellen** [comment]
updated

**Comment by dgeb** [comment]
thanks @tkellen!

**Pull request merged**